### DISCUSSION TITLE
Login 404 error both on Vercel and Localhost

### DISCUSSION THREAD

**Discussion by alexis2m** [avitorio/outstatic#219] [Q&A]
After following the "Add to Next.JS" guide, I cannot login as I get a 404 error for the following URL: `/api/outstatic/login`

Here's what's inside my `app/api/outstatic/route.ts`:

```
import { OutstaticApi } from 'outstatic'
 
export const GET = OutstaticApi.GET
 
export const POST = OutstaticApi.POST
```

But somehow, I cannot make it work. Here are my .env settings:

OST_REPO_OWNER=alexis2m
OST_REPO_SLUG=XXX
OST_GITHUB_ID=XXX
OST_GITHUB_SECRET=XXX

In Github, callback is correctly set to mywebsite.com/api/outstatic/callback

**Comment by avitorio** [comment]
Hey @alexis2m, sorry for the delay here. The api folder is incorrect. Please, visit the documentation and copy the correct path. It should be app/api/outstatic/[[...ost]]/route.ts or something like that. You missed the [[...ost]] part.
